src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -921,7 +933,7 @@ private async Task CalculateCommitMessage(UpdateAssetsParameters update, List<De
         ///     Because PRs tend to be live for short periods of time, we can put more information
         ///     in the description than the commit message without worrying that links will go stale.
         /// </remarks>
-        private async Task CalculatePRDescription(UpdateAssetsParameters update, List<DependencyUpdate> deps, List<GitFile> committedFiles, StringBuilder description)
+        private async Task CalculatePRDescription(UpdateAssetsParameters update, List<DependencyUpdate> deps, List<GitFile> committedFiles, StringBuilder description, List<DependencyDetail> allDependencies)
```

Comment:
Do we actually need allDependencies? It looks like we have all we need already in `deps`